vspace/src/main/webapp/WEB-INF/views/staff/modules/sequences/sequence.jsp

```diff
@@ -1,23 +1,200 @@
+            $("#cancelEditDescription").hide()
+            $("#editDescription").show()
+            var val = $("#newDescription").val();
+            $('<p id="description"style="margin-top: .5rem; margin-bottom: .5rem;">val</p>').insertBefore( "#newDescription" );
+            $("#newDescription").remove();
+            $("#description").text(val);
+        },
+        error: function(data) {
+            $(".open").removeClass("open");
+                var alert = $('<div class="alert alert-danger alert-dismissible fade show" role="alert"><p>We are sorry but something went wrong. Please try again later.</p><button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button></div>');
+                $('.error').append(alert);
+            }
+        });
+        
+   });
+    
+    // ---- Cancel Edited Description ----
+    
+    $("#cancelEditDescription").click(function(){
+        $("#submitDescription").hide()
+        $("#editDescription").show()
+        $("#cancelEditDescription").hide()
+        var description = document.getElementById("description").value
+        $('<p id="description" style="margin-top: .5rem; margin-bottom: .5rem;">description</p>').insertBefore( "#newDescription" );
+        $("#newDescription").remove()
+        $("#description").text(description);
+        
+    });
+    
+    // ------- Edit/Save/Cancel Description ends --------
+    
+    
+    
+    //------- edit title of Sequence starts --------
+    
+    $("#submitTitle").hide();
+    $("#cancelEditTitle").hide();
+    
+    	$("#editTitle").click(function() {
+
+    	var getTitleText = $("#title").text().split(": ")[1].trim()
+        $('<div class="col-4"><input id="newTitle" class="form-control" type="text"></div>').insertAfter( "#title" );
+    	document.getElementById("title").value = getTitleText;
+    	$('#title').text('Sequence: ')
+        $("#newTitle").val(getTitleText)
+        $("#editTitle").hide()
+        $("#submitTitle").show()
+        $("#cancelEditTitle").show()
+        
+    });
+    	
+    	
+    	
+    // ------ Submit title Sequence -------
+    	
+    $("#submitTitle").click(function() {
+        var formData = new FormData();
+        formData.append('title', $("#newTitle").val());
+        $.ajax({
+            url: "<c:url value="/staff/module/${module.id}/sequence/${sequence.id}/edit/title?${_csrf.parameterName}=${_csrf.token}" />",
+            type: 'POST',
+            cache       : false,
+            contentType : false,
+            processData : false,
+            data: formData,
+            enctype: 'multipart/form-data',
+            success: function(data) {
+                // replace text box with new description
+                $("#submitTitle").hide()
+                $("#cancelEditTitle").hide();
+                $("#editTitle").show()
+                var val = $("#newTitle").val();
+                $("#newTitle").closest('div').remove();
```

Comment:
why don't you give the div an id and remove it by id? much less error prone, because what if we need some extra layouting or something and need to add another div around the input field?